modules/@apostrophecms/login/index.js

```diff
@@ -158,36 +158,6 @@ module.exports = {
             await destroySession();
           }
         },
-        async setContext(req) {
-          if (!(self.apos.i18n.isValidLocale(req.body.locale) && [ 'draft', 'published' ].includes(req.body.mode))) {
-            throw self.apos.error('invalid');
-          }
-          const locale = req.body.locale;
-          const mode = req.body.mode;
```

Comment:
Are we going to need to bring this right back when we have second chance context switch? or will that be handled elsewhere

Reply:
No, the second chance switch is in this PR, see below.